tests/legacy-cli/e2e/tests/commands/cache/cache-info.ts

```diff
@@ -0,0 +1,63 @@
+import { execAndWaitForOutputToMatch } from '../../../utils/process';
+import { updateJsonFile } from '../../../utils/project';
+
+export default async function () {
+  const originalCIValue = process.env['CI'];
+
+  try {
+    // Should be enabled by default for local builds.
+    process.env['CI'] = '0';
+    await configureCacheOptions();
+    await execAndWaitForOutputToMatch(
+      'ng',
+      ['cache', 'info'],
+      /Effective status on current machine: enabled/,
+    );
+
+    // Should be enabled by default for CI builds.
```

Comment:
The expectation is for this to be "disabled", should the comment be updated?

Reply:
Yes